Content.Server/Sticky/Systems/StickySystem.cs

```diff
@@ -20,17 +21,48 @@ public sealed class StickySystem : EntitySystem
     [Dependency] private readonly SharedHandsSystem _handsSystem = default!;
     [Dependency] private readonly SharedInteractionSystem _interactionSystem = default!;
     [Dependency] private readonly SharedAppearanceSystem _appearance = default!;
+    [Dependency] private readonly EntityLookupSystem _lookup = default!;
 
     private const string StickerSlotId = "stickers_container";
 
     public override void Initialize()
     {
         base.Initialize();
+        SubscribeLocalEvent<TransferStickySurfaceOnSpawnBehaviorComponent, OnSpawnFromSpawnEntitiesBehaviourEvent>(TransferStickySurfaceFromDestroyedEntity);
+
+        SubscribeLocalEvent<StickyComponent, MapInitEvent>(OnMapInit);
         SubscribeLocalEvent<StickyComponent, StickyDoAfterEvent>(OnStickFinished);
         SubscribeLocalEvent<StickyComponent, AfterInteractEvent>(OnAfterInteract);
         SubscribeLocalEvent<StickyComponent, GetVerbsEvent<Verb>>(AddUnstickVerb);
     }
 
+    private void OnMapInit(EntityUid uid, StickyComponent component, MapInitEvent args)
+    {
+        if (!component.StickOnStart || component.StuckTo != null)
+            return;
+        
+        var stuck = false;
+        foreach (var entity in _lookup.GetEntitiesIntersecting(Transform(uid).MapPosition))
+        {
+            if (TryStickToEntity(uid, entity, null, component))
+            {
+                component.StickOnStart = false;
```

Comment:
i assume this is 'cuz mapinit can run again, but i don't think this is necessary after the other review, since this would never run if StuckTo has a value

Reply:
This one contradictory to previous review so I assume that you read code in that review wrong.

There could be situations where it MapInit will run again but `StuckTo` will be null. Atleast because posters can be unstuck after being stick on first map init.